x-pack/plugin/sql/sql-cli/src/test/java/org/elasticsearch/xpack/sql/cli/ConnectionBuilderTests.java

```diff
@@ -127,7 +129,12 @@ protected void checkIfExists(String name, Path p) {
             }
         };
         UserException actual = expectThrows(UserException.class, () ->
-            connectionBuilder.buildConnection("https://user@foobar:9242/", "keystore_location"));
+            buildConnection(connectionBuilder, "https://user@foobar:9242/", "keystore_location"));
         assertSame(actual, ue);
     }
+    
+    private ConnectionConfiguration buildConnection(ConnectionBuilder builder, String connectionStringArg,
+            String keystoreLocation) throws UserException {
```

Comment:
I think we usually align to the 1st parameter of the method: `ConnectionBuilder builder`.